src/native/external/zlib/trees.c

```diff
@@ -721,7 +721,7 @@ local void scan_tree(s, tree, max_code)
         if (++count < max_count && curlen == nextlen) {
             continue;
         } else if (count < min_count) {
-            s->bl_tree[curlen].Freq += count;
+            s->bl_tree[curlen].Freq += (ush)count;
```

Comment:
What's `ush`? Having trouble finding its definition.

Reply:
It's from a typedef here:

https://github.com/dotnet/runtime/blob/f750bb07795f2acfc48a3054ffdd3cf739dfbae4/src/native/external/zlib/zutil.h#L39

Reply:
Thanks. Oof, I really dislike cryptic c typedefs.

Reply:
Agreed. It's trying to match the maintainer's style as closely as possible, which is why it could look strange from our eyes.